test/node-test.js

```diff
@@ -505,11 +505,11 @@ describe('Node', function() {
       id: '1'
     }, {});
 
-    assert.typeOf(json.result, 'object');
-    assert.typeOf(json.result.curtime, 'number');
-    assert.typeOf(json.result.mintime, 'number');
-    assert.typeOf(json.result.maxtime, 'number');
-    assert.typeOf(json.result.expires, 'number');
+    assert(typeof json.result === 'object');
+    assert(Number.isInteger(json.result.curtime));
+    assert(Number.isInteger(json.result.mintime));
+    assert(Number.isInteger(json.result.maxtime));
+    assert(Number.isInteger(json.result.expires));
```

Comment:
Good call on making these tests more specific, they should only be integers

Reply:
Yeah, and not this:
```
> typeof NaN
'number'
```